tools/lint-md/lint-md.src.mjs

```diff
@@ -35,16 +48,36 @@ paths.forEach(async (path) => {
   const fileContents = file.toString();
   const result = await linter.process(file);
   const isDifferent = fileContents !== result.toString();
+  let manPageContent;
+  let isManPageModified = false;
+  if (resolve(path) === cliPath) {
+    try {
+      child_process.execFileSync(process.execPath, [
+        npxPath,
+        '--yes',
+        'github:nodejs/api-docs-tooling',
+        '-i', path,
+        '-o', '.tmp.1',
+        '-t', 'man-page',
+      ]);
+      manPageContent = fs.readFileSync('.tmp.1', 'utf-8');
+      isManPageModified = manPageContent !== fs.readFileSync(manFilePath, 'utf-8');
+      fs.rmSync('.tmp.1');
```

Comment:
Instead of reading a file to delete it right away, why not use stdout? Also using sync operations seems unwise, and re-reading the same man file in the loop looks like a mistake
```suggestion
      const cp = child_process.spawn(process.execPath, [
        npxPath,
        '--yes',
        'github:nodejs/api-docs-tooling',
        '-i', path,
        '-o', '-',
        '-t', 'man-page',
      ], { stdio: ['inherit', 'pipe', 'inherit']});
      manPageContent = Buffer.concat(await cp.stdout.toArray());
      isManPageModified = manPageContent.compare(actualManPageContent);
```

Reply:
Yes. If <https://github.com/nodejs/api-docs-tooling/pull/137> lands, this will be a lot simpler, as `child_process` won't be needed at all.